doc/08_api-overview.md

```diff
@@ -167,26 +162,26 @@ The underlying storage for voxels. Also used to pass voxels between functions.
 
 ## Supporting Classes
 
-### VoxelIsoSurfaceTool
-Provides functions to add or remove smooth terrain. Currently supported shapes are sphere, plane, cube, heightmap.
+### VoxelTool
+Provides functions to add or remove smooth terrain. The interface provides for points, lines, circles, spheres, boxes, and custom shapes, though not all are implemented.
 
-* You can generate a 2D heightmap and ask the tool to "rasterize" that portion of the world in a VoxelBuffer, which is what you will have to fill when implementing a custom VoxelStream.
+* You can generate a 2D heightmap and ask the tool to "rasterize" that portion of the world in a VoxelBuffer, which is what you will have to fill when implementing a custom VoxelGenerator.
 
-* Since Godot does not support 3D images, a custom shape could be added using an existing VoxelBuffer as input. This is not yet implemented.
+* Since Godot does not support 3D textures, a custom shape could be added using an existing VoxelBuffer as input. This is not yet implemented.
 
 
-### VoxelBoxMover
-Provides Axis Aligned Bounding Box based collision. (blocky only)
+### VoxelBoxMover (blocky)
+Provides Axis Aligned Bounding Box based collision.
 
 
-### Voxel
-A basic voxel unit. Creating Voxels with VoxelLibrary is recommended.  (blocky only)
+### Voxel (blocky)
+A basic voxel unit. Creating Voxels with VoxelLibrary is recommended.
 
 * Stores ID, name, material, transparency and type
 
 
-### VoxelLibrary
-A factory for creating Voxels. (blocky only)
+### VoxelLibrary (blocky)
+A factory for creating Voxels.
```

Comment:
A collection of voxel types, which is used for blocky only, similar to Minecraft or GridMap.

Reply:
replaced.